pennylane/measurements/shots.py

```diff
@@ -175,6 +175,18 @@ def __repr__(self):
         """The representation of the class"""
         return f"Shots(total_shots={self.total_shots}, shot_vector={self.shot_vector})"
 
+    def __eq__(self, other):
+        """Equality between Shot instances."""
+        return self.total_shots == other.total_shots and set(self.shot_vector) == set(
+            other.shot_vector
+        )
```

Comment:
```suggestion
        return self.total_shots == other.total_shots and self.shot_vector == other.shot_vector
```

`Shots((1,2))` should not equal `Shots((2,1))`

Reply:
Got it 👍🏼